pom.xml

```diff
@@ -244,6 +244,56 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+
```

Comment:
I don't think it's necessary to include jersey in the parent pom.xml, which will cause all modules to include jersey by default. The jersey1.x version is not recommended to be used, because it currently relies on eureka, and has to use the jersey1.x version

Reply:
in this place, it's just for dependencemanage to unfied the version.    the include action is in linkis-module module where include the jersey jars